app/views/posts/show.html.erb

```diff
@@ -1 +1,26 @@
-<p> Here is a list of all posts: </p>
\ No newline at end of file
+<% begin %>
+  Here is a list of post by user ID:
+<% end %>
+
+<div class="comment-wrapper">
+  <div class="comment-head">
+    <h1 class="post-title">
+      <%= @post.title %> By <%= @post.author.name %>
+    </h1>
+    <div class="comments-likes">
+      <p>comments: <%= @post.comments_counter %></p>
+      <p>likes: <%= @post.likes_counter %></p>
+    </div>
+  </div>
+
+  <p><%= @post.text %></p>
+
+  <div class="comments">
+    <% @comments.each do |comment| %>
+      <div class="single-comment">
+        <span><%= comment.author.name %> : <%= comment.text %></span>
```

Comment:
- [ ] For a better user experience, Kindly make sure you stick to the given template.

### YOur structure :point_down: 
![image](https://github.com/saluumaa/blog-app/assets/97100091/c8b08e69-4255-4ed4-9586-0d17c6dfeba9)

### The original template :point_down: 
![image](https://github.com/saluumaa/blog-app/assets/97100091/1fea4616-8061-4d86-8525-c121f9b4b94a)